lib/elixir/test/elixir/module/checker_test.exs

```diff
@@ -22,6 +22,7 @@ defmodule Module.CheckerTest do
           defmacro d, do: b()
           @deprecated "oops"
           def e, do: :ok
+          @callback f() :: :ok
         end
         """
       }
```

Comment:
What do you think about adding a `b.ex` here testing `behaviour_info` is added to "exports" with `@macrocallback`?